### DISCUSSION TITLE
What "switch sides" does?

### DISCUSSION THREAD

**Discussion by mikekgr** [rustdesk/rustdesk#5321] [Q&A]
Dear Sirs,
sorry if this information exist in documentation, I did not find something, so can somebody tell me what does the switch sides option?
Thanks and Best Regards,
Mike Kranidis

**Comment by 21pages** [comment]
change connection direction
![1691643119755](https://github.com/rustdesk/rustdesk/assets/14891774/e07a4326-e3f1-4c87-bba7-4d6e2699044e)

**Comment by SergeyMy** [comment]
just click on it, (in active connection) :):):):)

**Reply by mikekgr** [reply]
Thanks a lot guys !

**Reply by SergeyMy** [reply]
"switch sides"
 must be taken literally! who rules and who is ruled. :)
